scripts/lint.js

```diff
@@ -9,7 +9,9 @@ const prettierConfig = require('../config/prettier/prettierConfig');
 const prettierDefaultPath = require('../config/prettier/defaultPath');
 const eslintDefaultPath = require('../config/eslint/defaultPath');
 
-const args = process.argv.slice(2);
+const args = process.argv.slice(3);
```

Comment:
Am I right in thinking this PR could be reduced to these two `slice` lines? Do we need to keep the other changes?